examples/http-sse-service/http_sse_service.bal

```diff
@@ -0,0 +1,25 @@
+import ballerina/http;
+import ballerina/lang.runtime;
+import ballerina/random;
+
+service /stocks on new http:Listener(9090) {
+    resource function get .() returns stream<http:SseEvent, error?> {
+        StockPriceEventGenerator generator = new;
+        return new (generator);
+    }
+}
+
+class StockPriceEventGenerator {
+    int i = 0;
```

Comment:
Shall we rename this variable to something meaningful like `resetCounter` ?

Reply:
updated with [bc20a34](https://github.com/ballerina-platform/ballerina-distribution/pull/5533/commits/bc20a340e973a2fe6bd0cfff03292ead9b80cfe5)